app/views/docs/rest.phtml

```diff
@@ -1,12 +1,377 @@
+<h2><a href="/docs/rest#headers" id="headers">Headers</a></h2>
+<p>Appwrite's REST APIs expect certain headers to be included with each request:</p>
 
-<p>The specific structures used for Appwrite resources and their related endpoints can be found in the <b>References</b> section for each service.</p>
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 300px">Header</td>
+        <td style="width: 80px"></td>
+        <td>Description</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><code>X-Appwrite-Project: [PROJECT-ID]</code></td>
+        <td><span class="tag red">required</span></td>
+        <td>Provides the ID of your Appwrite Project to the REST API.</td>
+    </tr>
+    <tr>
+        <td><code>Content-Type: application/json</code></td>
+        <td><span class="tag red">required</span></td>
+        <td>Declares content type of the HTTP request.</td>
+    </tr>
+    <tr>
+        <td><code>X-Appwrite-Key: [API-KEY]</code></td>
+        <td><span class="tag red">optional</span></td>
+        <td>Used for authentication in server integrations. <b>Do not use API keys in client applications.</b></td>
+    </tr>
+    <tr>
+        <td><code>X-Appwrite-Response-Format: [VERSION-NUMBER]</code></td>
+        <td><span class="tag red">optional</span></td>
+        <td>Used for backwards compatibility. The response will be formatted to be compatible the provided version number.</td>
+    </tr>
+    <tr>
+        <td><code>X-Fallback-Cookies: [FALLBACK-COOKIES]</code></td>
+        <td><span class="tag red">optional</span></td>
+        <td>Used in scenarios where browsers do not allow third-party cookies. Often used when there is no <a href="/docs/custom-domains">Custom Domain</a>.</td>
+    </tr>
+    </tbody>
+</table>
 
-<h2><a href="/docs/realtime#authentication" id="authentication">Authentication</a></h2>
-<p>The REST API authenticates using Appwrite accounts and sessions. Both accounts and sessions can be created with REST using the <b>POST</b> <b>/v1/account</b>, <b>/v1/account/sessions/email</b>, <b>/v1/account/sessions/anonymous</b>, or <b>/v1/account/sessions/phone</b> endpoints.</p>
+<h3><a href="/docs/rest#no-headers" id="no-headers">Using Appwrite Without Headers</a></h3>
+<p>Some use cases do not allow the use of custom headers, such as when embedding images from Appwrite in HTML. In these cases, you can provide the Appwrite project ID using the query parameter <code>project</code>.</p>
+<div class="ide margin-bottom" data-lang="html" data-lang-label="HTML">
+    <pre class="line-numbers"><code class="prism language-html" data-prism><?php echo htmlentities('<img src="https://[HOSTNAME_OR_IP]/v1/storage/buckets/[BUCKET_ID]/files/[FILE_ID]/preview?project=[PROJECT_ID]">'); ?></code></pre>
```

Comment:
This is displaying a blank code example

Reply:
This is also showing up when I test it 🤔

Reply:
<img width="804" alt="Screen Shot 2022-12-20 at 12 22 50 PM" src="https://user-images.githubusercontent.com/29069505/208727909-52413e99-1cd7-4cd5-933e-cc7e6ec1b79c.png">

Reply:
Resolved on my end :+1: